### DISCUSSION TITLE
In-Game Microphone Support

### DISCUSSION THREAD

**Discussion by Silas888** [LizardByte/community#129] [Sunshine]
I'm curious if in-game microphone support will be considered 
being added to the roadmap?

**Comment by gihov** [comment]
I would love this feature too !

**Comment by KINGLIFER** [comment]
The bulk of the original Nividia crew would love this too since they more than likely have the Joystick that has the built in Mic. I have actually had this work by pluggin in a headset into the mic. It actually works but the joystick mic does not.

**Comment by tmplt** [comment]
I'll add my vote to have this in-scope: it would allow me to throw my noisy computer into the basement.

**Comment by michaelcurry** [comment]
Does anyone know if there has been any movement on this feature request?

**Reply by ReenigneArcher** [reply]
It's on the roadmap: https://github.com/LizardByte/roadmap/issues/56
